service/src/LevelZero.cpp

```diff
@@ -603,6 +707,34 @@ namespace geopm
                         ": Sysman failed to set frequency.", __LINE__);
     }
 
+    void LevelZeroImp::performance_factor_control(int geopm_domain,
```

Comment:
Similar interface change suggestion as for other ``performance_factor`` functions applies here as well.